Helper/MerchantMetrics.php

```diff
@@ -0,0 +1,343 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2018 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Helper;
+
+use http\Encoding\Stream;
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+use Bolt\Boltpay\Helper\Config as ConfigHelper;
+use Magento\Framework\Filesystem\DirectoryList;
+use Magento\Store\Model\StoreManagerInterface;
+use Bolt\Boltpay\Helper\Log as LogHelper;
+
+
+
+/**
+ * Boltpay Merchant Metrics wrapper helper
+ *
+ * @SuppressWarnings(PHPMD.TooManyFields)
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
```

Comment:
are these valid warnings? Ideally we don't ignore warnings in new code or remove this rule all together via some config